Add misusage description of token (#109)
ietf-teep/teep-protocol: draft-ietf-teep-protocol.md

  generated by a TAM MUST be randomly created.

>   Consequent token values MUST be different for each request message

Reviewer: ```suggestion
  Subsequent token values MUST be different for each request message
```

  to distinguish the correct response from multiple requests.
  The token value MUST NOT be used for other purposes, such as a TAM to
  identify the devices and/or a device to identify TAMs or Trusted Components.
  The TAM MUST expire the token value after receiving the first responce

>   from the device and ignore any consequent messages which have the same token

Reviewer: ```suggestion
  from the device and ignore any subsequent messages that have the same token
```
